tests/splinter_test.py

```diff
@@ -204,6 +204,33 @@ def test_add_two_assets_upload(self):
             self.assertEqual(assets[1]['mimetype'], u'image')
             self.assertEqual(assets[1]['duration'], settings['default_duration'])
 
+    def test_add_asset_streaming(self):
+        with Browser() as browser:
+            browser.visit(main_page_url)
+
+            wait_for_and_do(browser, '#add-asset-button', lambda btn: btn.click())
+            sleep(1)
+
+            wait_for_and_do(browser, 'input[name="uri"]', lambda field: field.fill('rtmp://controlstreams.com:1935/vtvmoviles3/vtvmoviles3'))
+            sleep(1)
+
+            wait_for_and_do(browser, '#add-form', lambda form: form.click())
+            sleep(1)
```

Comment:
I guess this is not something we can work on right now but we really should update `wait_for_and_do` to micro sleep until the element is available (or there's a timeout) so we can remove all static `sleep` (which slow down the unit tests a lot more than necessary).